src/main/java/net/spy/memcached/ConnectionFactory.java

```diff
@@ -159,7 +159,7 @@ MemcachedNode createMemcachedNode(SocketAddress sa, SocketChannel c,
   HashAlgorithm getHashAlg();
 
   /**
-   * Maximum number of milliseconds to wait between reconnect attempts.
+   * Maximum number of seconds to wait between reconnect attempts.
```

Comment:
I think this should be `milliseconds.`

Because, in this commit, you just used `f.getMaxReconnectDelay()` 
and as you already know `maxDelay` is applied to calculate `delay` 

`long delay = (long) Math.min(maxDelay, Math.pow(2,
        node.getReconnectCount()) * 1000);
    long reconnectTime = System.currentTimeMillis() + delay;`

As we can see above, `delay` will be added to `currentTimeMillis()` 
and the value `1000` is multiplied only to `Math.pow(2,
        node.getReconnectCount())` not to `maxDelay`.

So I think it should be milliseconds as originally it was.

Reply:
although maxDelay is in millisecond, `maxDelay = TimeUnit.SECONDS.toMillis(f.getMaxReconnectDelay());`  MaxReconnectDelay should be in second